### DISCUSSION TITLE
blog/como-criar-um-plugin-spigot-em-kotlin

### DISCUSSION THREAD

**Discussion by giscus** [doceazedo/doce.sh#14] [Comments]
# blog/como-criar-um-plugin-spigot-em-kotlin

Aprenda a como criar o projeto de um plugin Spigot usando Kotlin

https://doceazedo.com/blog/como-criar-um-plugin-spigot-em-kotlin

**Comment by murilonerdx** [comment]
Nossa muito top mano!!
